xmonad-contrib.cabal

```diff
@@ -25,15 +25,17 @@ extra-source-files: README.md CHANGES.md scripts/generate-configs scripts/run-xm
                     scripts/window-properties.sh
                     scripts/xinitrc scripts/xmonad-acpi.c
                     scripts/xmonad-clock.c
-                    tests/genMain.hs
+                    tests/Main.hs
+                    tests/Utils.hs
+                    tests/Instances.hs
```

Comment:
Do we need these now that we have a test-suite with `hs-source-dirs: tests`?

Reply:
I think we do, or at least that's what I found in the core library https://github.com/xmonad/xmonad/blob/96d4f1fe855c07538ea5833f89a8043ac2224515/xmonad.cabal#L41

Reply:
Those lines predate [adding other-modules to the test-suite](https://github.com/xmonad/xmonad/commit/0c97a8975425c525598ef0e3f3e1bfc84f779b1a#diff-5911ea1a0e2ec2cb41389429db836564c3df03359044f63df35d1b0a59ff576dR134).

I mean, it may be required, I am no expert on Cabal. It just seems somewhat unlikely. Can you perhaps try if cabal/stack sdist works properly without it?

Reply:
Removing them doesn't break anything. I'll push the change

Reply:
Thanks :-)

Reply:
Awesome :smile: